packages/sui-ssr/build-device.js

```diff
@@ -0,0 +1,8 @@
+export const buildDeviceFrom = ({request = {}, window = {}}) => {
```

Comment:
IMO something like `buildDeviceInfo`, `getDeviceData` or even just `buildDevice` would be more descriptive

Reply:
The idea is combine the name of the method with the param name, to get something like: `buildDeviceFrom({window})`

To achive a more semantic method call